lib/pivotal-tracker/proxy.rb

```diff
@@ -0,0 +1,17 @@
+class PivotalTracker::Proxy
```

Comment:
It is a general purpose Proxy. Any nested accounts can use to to called chained methods